packages/dashboard/src/customization/propertiesSections/settings/section.tsx

```diff
@@ -43,17 +59,45 @@ export const SettingsSection = ({
             className='settings-property-label'
             style={{ gap: awsui.spaceScaledS }}
           >
-            <label htmlFor='decimal-places'>Decimal places</label>
-            <div className='settings-property-label-control'>
-              <Input
-                type='number'
-                data-testid='decimal-place-config'
-                controlId='decimal-places'
-                ariaLabel='decimal places'
-                value={significantDigits?.toFixed() ?? ''}
-                onChange={onSignificantDigitsChange}
-              />
-            </div>
+            <label htmlFor='decimal-places'>
+              <b>Places</b>
+            </label>
+            <Controller
+              control={control}
+              name='decimalPlaces'
+              rules={{
+                max: {
+                  value: 100,
+                  message: 'Decimal places must be between 0 and 100.',
+                },
+              }}
+              render={({ field, fieldState }) => (
+                <FormField
+                  errorText={fieldState.error?.message}
+                  constraintText='Must be between 0 and 100.'
+                >
+                  <div className='settings-property-label-control'>
+                    <Input
+                      type='number'
+                      data-testid='decimal-place-config'
+                      controlId='decimal-places'
+                      ariaLabel='decimal places'
+                      value={decimalPlacesValue?.toFixed() ?? ''}
+                      onKeyDown={(e) =>
+                        ['-', '+'].includes(e.detail.key) && e.preventDefault()
+                      }
+                      onChange={({ detail: { value } }) => {
+                        if (value.startsWith('-')) field.onChange(undefined);
```

Comment:
Instead of ignoring the input of "-", could we use the minimum validation error?

Reply:
yes done.